company_website/constants.py

```diff
@@ -5,8 +5,8 @@ class EstimateConstants:
     product_choices = (("build", "Build something new"), ("improve", "Improve existing project"))
     budget_choices = (
         ("-8", "up to $8.000"),
-        ("8-25", "$8.000 - $25.000"),
-        ("25-100", "$25.000 - $100.000"),
+        ("8-25", "$8.000 — $25.000"),
+        ("25-100", "$25.000 — $100.000"),
```

Comment:
What's the difference here?

Reply:
It's super subtle here, I know 😉 
We had **hyphen** before and now it's an **em dash**. You can hardly see it here, but it's an entirely different punctuation mark much longer than the hyphen.
`-`  <-  hyphen
`—`  <-  em dash